crates/forge/tests/it/test_helpers.rs

```diff
@@ -209,6 +209,7 @@ impl ForgeTestData {
             .with_cheats_config(CheatsConfig::new(&config, opts.clone(), Some(artifact_ids), None))
             .sender(config.sender)
             .with_test_options(self.test_opts.clone())
+            .enable_isolation(config.isolate)
```

Comment:
Does not work yet as expected, unsure why

Reply:
What's the issue?

Reply:
It turns out the isolate option exists as part of the evm_opts and the way it gets cloned with defaults it would override that what was set it the config. Something that could be improved as part of https://github.com/foundry-rs/foundry/issues/7322